gobiko/apns/client.py

```diff
@@ -180,20 +181,14 @@ def _send_message(self, registration_id, alert,
 
         auth_token = auth_token or self._create_token()
 
-        if not topic:
-            topic = bundle_id if bundle_id else self.bundle_id
```

Comment:
The expression `topic or bundle_id or self.bundle_id` is logically duplicated in two places: in the check at the beginning of the function and in the middle of the function, where it is actually used. I think it is better to move `topic` evaluation to the beginning, where it is validated that at least one source of its value is available